.github/workflows/release-build.yml

```diff
@@ -1,12 +1,16 @@
 name: Build release artifacts
 
 on:
-  # Handles a 'helics-release-build' dispatch event type
-  # The event payload should contain these fields:
-  # packages - a list of which packages to build (everything, archive, msvc, installer, sharedlib)
-  # commit (optional) - the commit-ish to do a release build with
-  repository_dispatch:
+  # Picking a workflow branch that is a tag, in combination with using the commit field is likely to fail for magic git clone/checkout reasons
   workflow_dispatch:
+    inputs:
+      packages:
```

Comment:
where does this input get specified?  Is this a manual option?  if so are the listed options here all the recognized ones?

Reply:
It shows up when you go to manually trigger a GitHub workflow (only shows up when the workflow file in the default/main branch has a list of inputs). See https://github.com/GMLC-TDC/helics-action/actions/workflows/test-helics-install.yml for an example.

The options listed are the words that will be recognized.